spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerPageableQueryOptions.java

```diff
@@ -68,4 +70,16 @@ public SpannerPageableQueryOptions setAllowPartialRead(boolean allowPartialRead)
 		super.setAllowPartialRead(allowPartialRead);
 		return this;
 	}
+
+	@Override
+	public SpannerPageableQueryOptions setTimestamp(Timestamp timestamp) {
+		super.setTimestamp(timestamp);
+		return this;
+	}
+
+	@Override
+	public SpannerPageableQueryOptions setBoundedTimestamp(boolean boundedTimestamp) {
```

Comment:
I think changing from `boolean` to an actual `BoundedTimestamp` would be more intuitive.

Reply:
@dmitry-s @meltsufin  done. PTAL